AsyncDisplayKit/ASTableView.mm

```diff
@@ -903,6 +915,21 @@ - (void)rangeController:(ASRangeController *)rangeController didDeleteSectionsAt
     return; // if the asyncDataSource has become invalid while we are processing, ignore this request to avoid crashes
   }
 
+  if (_contentOffsetAdjustmentTopVisibleRow != nil) {
+    __block NSInteger adjustedSection = _contentOffsetAdjustmentTopVisibleRow.section;
+    __block NSInteger adjustedRow = _contentOffsetAdjustmentTopVisibleRow.row;
+    [indexSet enumerateIndexesUsingBlock:^(NSUInteger idx, BOOL *stop) {
+        if (adjustedSection > idx) {
+          adjustedSection--;
+        } else if (adjustedSection == idx) {
+          adjustedRow = 0;
```

Comment:
cc @zintus - I'd love to get this landed.  Do you have any thoughts on @eanagel's question here?

@maicki do you think we should land this without further testing?  Since it looks like we may not be able to get a test project, and this feature is not used by that many people, it's probably better to take the fix and rely on @zintus' own testing.